runtime/src/lib.rs

```diff
@@ -210,6 +210,7 @@ impl Filter<Call> for BaseFilter {
 
 			Call::XYK(_)
 			| Call::AssetRegistry(_)
+			| Call::Balances(_)
```

Comment:
will staking, governance etc. work?

Reply:
Yes, it should. It was tested previously with treasury tip.